.golangci.yml

```diff
@@ -69,4 +69,6 @@ issues:
       linters:
         - gosec
         - dupl
+    - text: 'shadow: declaration of "(err|ctx)" shadows declaration at'
+      linters: [ govet ]
```

Comment:
Here I just disable `err|ctx` shadowing for the `go vet` linter, other rules will work.
I added this rule since I want not to create ctx2..n and err2..n variables.

Reply:
Is okay for you?